GreenhouseNode/GreenhouseNode/temp/semaphore_handler.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * semaphore_handler.c
+ *
+ * Created: 12/11/2021 20.01.49
+ *  Author : Michel Sofus Engelhardt Sommer, 273966
+ */
+
+#include "semaphore_handler.h"
+
+void semaphores_init() {
+
+	loraSemaphore = xSemaphoreCreateMutex();
+	if( loraSemaphore == NULL )
```

Comment:
This code seems a little funky, setting a variable them checking it and then setting it again.